### PR TITLE
Fix up package.json and README

### PR DESCRIPTION
**Pull request by srl295**
Fixes: https://github.com/IBM-Bluemix/gp-vscode-plugin/issues/1
* updated links to point to the new home and
* fixed some minor packaging issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,15 +9,15 @@
     "color": "#f2f2f2",
     "theme": "light"
   },
-  "license": "SEE LICENSE IN LICENSE.txt",
+  "license": "Apache-2.0",
```
Comment (srl295) [inline package.json]: from http://spdx.org/licenses/

File: README.md [hunk 2]
```diff
@@ -22,14 +22,14 @@ You can download your bundles from the editor and then select the language that
...
 
-+ If you are using Bluemix Code, then run **bluemix-code g11n-0.0.5.vsix** 
-- If you are using Visual Studio Code, then run **code g11n-0.0.5.vsix**
++ If you are using Bluemix Code, then run **bluemix-code g11n-x.y.z.vsix** 
+- If you are using Visual Studio Code, then run **code g11n-x.y.z.vsix**
 
 ## Requirements
 
-You must first create an instance of the Globalization Pipeline service on Bluemix before you can use this extension to work with translation bundles.
+You must first [create an instance of the Globalization Pipeline service on Bluemix](https://github.com/IBM-Bluemix/gp-common#quick-start-guide) before you can use this extension to work with translation bundles.
```
Comment (srl295) [inline README.md]: updated to use our common QSG

File: README.md [hunk 3]
```diff
@@ -22,14 +22,14 @@ You can download your bundles from the editor and then select the language that
 
 ## Install
 
-You can install the Globalization Pipeline extension by directly downloading the .vsix package from [releases](https://github.ibm.com/gp/gp-vscode-plugin/releases) and then from the command line do one of the following:
+You can install the Globalization Pipeline extension by directly downloading the .vsix package from [releases](https://github.com/IBM-Bluemix/gp-vscode-plugin/releases) and then from the command line do one of the following:
 
-+ If you are using Bluemix Code, then run **bluemix-code g11n-0.0.5.vsix** 
-- If you are using Visual Studio Code, then run **code g11n-0.0.5.vsix**
++ If you are using Bluemix Code, then run **bluemix-code g11n-x.y.z.vsix**
```
Comment (srl295) [inline README.md]: Updated to be future proof™

**Comment by steveatkin** [comment]
Looks good to me

**Pull request merged**